NewBlog/views.py

```diff
@@ -0,0 +1,69 @@
+from django.http import HttpRequest, HttpResponse
+
+
+def func_hello(response: HttpRequest) -> HttpResponse:
+    return HttpResponse('Hello Page')
+
+
+def func_empty(response: HttpRequest) -> HttpResponse:
+    return HttpResponse('Empty Page')
+
+
+def func_article(response: HttpRequest, article: str) -> HttpResponse:
```

Comment:
Не дуже вдала назва.
```suggestion
def article_detail_view(response: HttpRequest, article: str) -> HttpResponse:
```